lib/bundle/brew_dumper.rb

```diff
@@ -16,12 +16,18 @@ def self.formulae
     end
 
     def self.dump
+      bs = BrewServices.new
       formulae.map do |f|
+        if bs.started?(f[:full_name])
+          restart = ', service_restart: true'
+        else
+          restart = ''
+        end
```

Comment:
And access this from either `formulae` or `self.formula_inspector` so you can have it be e.g. `f[:service_started]`